stories/multi-select.stories.js

```diff
@@ -162,4 +162,37 @@ storiesOf("React Multi Select", module)
         showSelectAll={boolean("Show select all", true)}
       />
     );
+  })
+  .add("With custom components and custom value", () => {
+    class ValueController extends React.Component {
+      state = {
+        value: ""
+      };
+
+      onChange = value => {
+        console.log(value);
```

Comment:
This is not needed. Since the action("onChange") will fire this event and it will be shown in the storybook

Reply:
Just a remainder from my debugging. Removed.